utils/address.ts

```diff
@@ -0,0 +1,6 @@
+export const shortenAddress = (address: string) => {
+  if (address.length < 9) {
```

Comment:
Would this ever be true?

Reply:
It would only be true if a developer used this method incorrectly. Maybe we should throw inside this check instead.

Reply:
Why length of 9 though? That seems arbitrary. Shouldn't the check instead be that the address starts with `0zk` and the rest is valid base58? (or whatever format railgun addresses use)

Reply:
Yes, I think the 9 was supposed to be like minimum length of the string without repeating characters before we changed the slice parameters. I agree we should validate the the address. I will add that and throw an error if it is an invalid string